marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -134,4 +172,21 @@ private void addFieldAndIndexes() {
         systemEndIndex.put("range-value-positions", false);
         new DatabaseManager(getHubConfig().getManageClient()).save(newNode.toString());
     }
+
+    private InputStream asInputStream(String value) {
```

Comment:
This is only used once, and it's one line of code, so it's indirection that doesn't buy us anything - just put that one line of code in the one place where this method is invoked.

Reply:
Moved to the method